### ISSUE TITLE
'header' option is mandatory for Excel import

### ISSUE THREAD

**Issue by okennedy** [VizierDB/web-ui#273]
```
[info] 15:45:39.970 [qtp1880669668-3401] ERROR org.mimirdb.api.MimirVizierServlet - MimirAPI ERROR: 
[info] java.lang.IllegalArgumentException: Parameter "header" is missing in options.
[info] 	at com.crealytics.spark.excel.DefaultSource.checkParameter(DefaultSource.scala:68)
```
Whether or not the **File Includes Headers** is checked, this error appears.  Should be an easy fix.

**Comment by mrb24** [comment]
closed by commit: a6a35848215103c960194e20a9299b4b622fddde
if not specified, header now defaults to true for excel unload.

**Comment by mrb24** [comment]
and closed by commit: 7891e4aa152dd6468765d9f3b2adbf28b36285d7
header now set for excel load.
